flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBNativeMetricMonitor.java

```diff
@@ -54,7 +54,7 @@
 	private RocksDB rocksDB;
 
 	RocksDBNativeMetricMonitor(
-		@Nonnull RocksDB rocksDB,
+		RocksDB rocksDB,
```

Comment:
Why is this change required, looks problematic to me.

Reply:
There were problems with tests for me locally which passed null args here and bellow.
I will replace nulls with non-null stubs.